apps/verticalhome/js/app.js

```diff
@@ -199,6 +208,11 @@
         item.updateTitle();
       });
       this.renderGrid();
+
+      app.pinManager.items.forEach(function eachItem(item){
```

Comment:
onLocalized  function on event "localized".
This function is used for both screens (for more apps and for main screen).